website/docs/components/link/inline/partials/accessibility/accessibility.md

```diff
@@ -0,0 +1,24 @@
+<!-- TODO: Add conformance rating -->
+## Best Practices
+
+### Annotations in design
+
+Include annotations of the non-visual experience in your handoff notes. For example:
+
+![Annotation of focus order of the link](/assets/components/link/standalone/link-annotation.png =440x*)
+
+### Interactive transitions
+
+Animations or transitions will not take place if the user has `prefers-reduced-motion` enabled in their browser.
```

Comment:
I'm pretty sure there are no animations/transitions on the inline link, only the standalone link.

Reply:
Ah, yes! Copy pasta, thanks for catching that!